try:
    x = 5/0
except Exception as a:
    print(a)